### PR TITLE
Add comments and logging to pkg/cloud

### PR DESCRIPTION
**Pull request by thomasvn**
## What does this PR change?
* Update to comments and logging

## Does this PR relate to any other PRs?
* N/A

## How will this PR impact users?
* N/A

## Does this PR address any GitHub or Zendesk issues?
* N/A

## How was this PR tested?
* N/A

## Does this PR require changes to documentation?
* N/A

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next OpenCost release? If not, why not?
*

### CODE REVIEW

**Comment by thomasvn** [comment]
@Sean-Holcomb Could you validate accuracy of the comments?

**Review by nik-kc — APPROVED** [review]

**Comment by thomasvn** [comment]
Thanks @nik-kc for the review! I think this will need approval & merge from an OpenCost maintainer.

**Review by michaelmdresser — APPROVED** [review]

**Pull request merged**
